src/javascript/app/pages/trade/price.js

```diff
@@ -151,6 +156,21 @@ const Price = (() => {
             return;
         }
 
+        // hide all containers except current one
+        if (position !== 'middle') {
```

Comment:
can we switch this around so it becomes `if (position === 'middle') {...} else {...`